### ISSUE TITLE
Port FxCop rule CA2243: AttributeStringLiteralsShouldParseCorrectly

### ISSUE THREAD

**Issue by lgolding** [dotnet/roslyn-analyzers#554]
**Title:** Attribute string literals should parse correctly

**Description:**

The string literal parameter of an attribute does not parse correctly for a URL, a GUID, or a version.

**Dependency:** WordParser

**Notes:**

**Comment by bkoelman** [comment]
In VS2015 Update1, a violation of this rule is reported on:

```csharp
[assembly: AssemblyInformationalVersion("0.0.1-ci")]
```

which I believe is.... well, at least unfortunate. Because `InformationalVersion` is often used for [semantic versioning](http://semver.org/).

**Comment by lgolding** [comment]
@bkoelman  That sounds reasonable to me.

**Comment by basoundr** [comment]
#930 implements the analyzer

**Comment by bkoelman** [comment]
I believe this can be closed, now that the analyzer has been implemented.

**Comment by bkoelman** [comment]
Oh wait, it is tagged with `Needs-Fixer`.

What should a fixer do here?

**Comment by srivatsn** [comment]
Ya I don't know what fix we could offer - @basoundr did you have any specific fix in mind? If not, we should delete the fixer stubs and close the issue.

**Comment by basoundr** [comment]
We can close this. I did not have anything specific in my mind. I mistakenly added `Needs-Fixer` by habit

**Comment by srivatsn** [comment]
Can you send out a PR to delete the stubs and then close it out?

**Comment by basoundr** [comment]
I will

**Comment by bkoelman** [comment]
@basoundr Can you close this now?

**Comment by basoundr** [comment]
@bkoelman Thanks for reminding!
